src/components/connect.js

```diff
@@ -3,6 +3,7 @@ import storeShape from '../utils/storeShape'
 import shallowEqual from '../utils/shallowEqual'
 import wrapActionCreators from '../utils/wrapActionCreators'
 import warning from '../utils/warning'
+import isFunction from 'lodash/isFunction'
```

Comment:
It’s tiny but we already use `typeof x === 'function'` elsewhere, let’s keep doing it here as well?